src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/BasicExpressionCompilerTest.vbproj

```diff
@@ -19,6 +19,7 @@
   <ItemGroup Label="File References">
```

Comment:
Consider adding ` <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file.